static/app/utils/panelProvider.tsx

```diff
@@ -0,0 +1,19 @@
+import {ThemeProvider, useTheme} from '@emotion/react';
+
+const smallerBorderRadii = {
+  borderRadius: '4px',
+  borderRadiusBottom: '0 0 4px 4px',
+  borderRadiusTop: '4px 4px 0 0',
+  borderRadiusLeft: '4px 0 0 4px',
+  borderRadiusRight: '0 4px 4px 0',
+};
+
+const PanelProvider = ({children}: {children: React.ReactNode}) => {
+  const theme = useTheme();
+
+  return (
+    <ThemeProvider theme={{...theme, ...smallerBorderRadii}}>{children}</ThemeProvider>
```

Comment:
I think we want to put the theme value here in a `useMemo`, otherwise every single consumer will rerender when this one does.

Reply:
Good point! https://github.com/getsentry/sentry/pull/41205/commits/1a4eba415b717483ed11460132653ab6d3e175b2